src/test/resources/data/FeedbackResponseCommentsLogicTest.json

```diff
@@ -0,0 +1,1709 @@
+{
+  "accounts": {
```

Comment:
Why do we need a separate data file for this test? If you want to add new data for the new test then load the typical data and load the new data, then merge them

Reply:
The reason for not using `TypicalDataBundle` is that adding new data to `TypicalDataBundle` will break a lot of other tests for some reason.

Reply:
@hhdqirui I think you can take a look at `teammates/logic/core/FeedbackResponsesLogicTest.java` to check how it is using both `typicalDataBundle` and extra data bundle

Reply:
Thank you for the suggestion. I will look into that.